byzcoin/tx_pipeline.go

```diff
@@ -327,13 +330,26 @@ func (p *txPipeline) processTxs(txChan <-chan ClientTransaction, initialState *t
 		p.wg.Add(1)
 		defer p.wg.Done()
 		intervalChan := getInterval()
+		var txHashes [][]byte
+	leaderLoop:
 		for {
 			select {
 			case tx, ok := <-txChan:
 				if !ok {
 					log.Lvl3("stopping txs processor")
 					return
 				}
+				for _, txHash := range txHashes {
+					if bytes.Compare(txHash, tx.Instructions.Hash()) == 0 {
+						log.Lvl2("Got a duplicate transaction, ignoring it")
+						continue leaderLoop
```

Comment:
You don't need a label, you can just break the select.

Reply:
there is a `for` between here and the `select`. But I'll have a look at it if I can take it out with your previous comment.